examples/deutsch.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
You can remove this notice per: https://github.com/quantumlib/Cirq/blob/master/examples/README.rst

Reply:
Done.